TaskER_FlowChart.md

```
@@ -1,7 +1,28 @@
 # TaskER FlowChart Block Inventory
 
+<!-- markdownlint-disable MD024 - Duplicate headings (Parameters, Example, Entry Point, Behavior) are intentional for consistent block structure -->
+
 This document provides a visual inventory of TaskER workflow blocks with their corresponding parameters.
 
+## Table of Contents
+
+1. [Execution Block](#1.-execution-block)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)